functions/src/notifications/types.ts

```diff
@@ -70,3 +63,8 @@ export interface BillHistoryUpdateNotificationFields {
   }
   createdAt: FirebaseFirestore.Timestamp
 }
+
```

Comment:
Agreed that this is sub-ideal, but we have a separate block of tech debt being allocated for extracting shared types between components and functions, so I figure we can wait to solve this until we're ready to take that on.